databricks/koalas/tests/test_series.py

```diff
@@ -1894,3 +1895,49 @@ def test_tail(self):
             self.assert_eq(pser.tail(-1001), kser.tail(-1001))
             with self.assertRaisesRegex(TypeError, "bad operand type for unary -: 'str'"):
                 kser.tail("10")
+
+    def test_product(self):
+        pser = pd.Series([10, 20, 30, 40, 50])
+        kser = ks.from_pandas(pser)
+        self.assert_eq(pser.prod(), kser.prod())
+
+        # Containing NA values
+        pser = pd.Series([10, np.nan, 30, np.nan, 50])
+        kser = ks.from_pandas(pser)
+        self.assert_eq(isclose(pser.prod(), kser.prod()), True)
```

Comment:
Shall we use `self.assertTrue()`?

Reply:
btw, doesn't `self.assert_eq(pser.prod(), kser.prod(), almost=True)` work?

Reply:
Oh, It works. Thanks! 👍